crates/cext/src/transpiler/target.rs

```diff
@@ -468,13 +471,19 @@ impl TargetEntry {
             operation: StandardOperation::Gate(operation),
             params,
             map: Default::default(),
+            name: Some(operation.name().to_string()),
         }
     }
 
-    pub fn new_fixed(operation: StandardGate, params: SmallVec<[Param; 3]>) -> Self {
+    pub fn new_fixed(
+        operation: StandardGate,
+        params: SmallVec<[Param; 3]>,
+        name: *const c_char,
+    ) -> Self {
         Self {
             operation: StandardOperation::Gate(operation),
             params: Some(params),
+            name: Some(unsafe { CStr::from_ptr(name).to_str().unwrap().to_string() }),
```

Comment:
```suggestion
            name: Some(unsafe { CStr::from_ptr(name).to_str().expect("Error while extracting the given name.").to_string() }),
```
It's better to include a panic message like this whenever we do this operation so that we know which step went wrong.

Reply:
Added.